scripts/cmake/config/default.cmake

```diff
@@ -436,6 +436,13 @@ add_config(
     DESCRIPTION "Build unit test components"
 )
 
+add_config(
+    CONFIG_NAME ENABLE_BUILD_EFI
+    CONFIG_TYPE BOOL
+    DEFAULT_VAL ON
```

Comment:
I think we need to default this to OFF until EFI support becomes a little more stable